fe/fe-core/src/main/java/org/apache/doris/http/HttpServer.java

```diff
@@ -17,6 +17,17 @@
 
 package org.apache.doris.http;
 
+import io.netty.bootstrap.ServerBootstrap;
```

Comment:
remain the import order.

Reply:
We'd better to provide some config for popular IDE